h/models/token.py

```diff
@@ -39,8 +39,9 @@ class Token(Base, mixins.Timestamps):
     #: A NULL value in this column indicates a token that does not expire.
     expires = sqlalchemy.Column(sqlalchemy.DateTime, nullable=True)
 
-    def __init__(self, userid):
+    def __init__(self, userid, expires=None):
```

Comment:
Sure, but that can happen in a separate PR as this one isn't introducing the override of the constructor.